### PR TITLE
Support for 'stack output' command

### PR DESCRIPTION
**Pull request by mikocot**
This PR introduced support for `pulumi stack output` command in addition to the existing ones. 

The reasons for this change are explained in the following issue:
[https://github.com/pulumi/actions/issues/868](https://github.com/pulumi/actions/issues/868)

I've chosen `output` as a name for the command as it reflects the method called by CLI, although internally in the sdk it calls `stack.outputs()`.

As mentioned previously I have issues testing this solution, so I invite you to test it yourself and collaborate on the final solution. I have found no relevant tests nor I can see a good way to unit test it, but I've updated the test workflow.

### CODE REVIEW

File: src/main.ts
Comment (RobbieMcKinstry) [inline src/main.ts:92]: Was this line added as part of the debugging process? I'm not totally able to make sense of it's intent. Forgive me if it should be obvious. 

I'm going to checkout this PR locally and see if I can add a test. I'll take a closer look locally.

**Review by RobbieMcKinstry — CHANGES_REQUESTED** [review]
Just one question; otherwise, it looks good. I think `output` captures the command's functionality. I'd like a test and additional documentation in the README, but otherwise it looks good.

I'm going to check out this code locally and see if I can add a test.

**Comment by RobbieMcKinstry** [comment]
Oh, also, it looks like this PR requires a rebase before we can merge it.

File: src/main.ts
Comment (mikocot) [reply src/main.ts:92]: yeah, probably, it's been a while :)

**Comment by mikocot** [comment]
The PR has been reopened as #1002 

https://github.com/pulumi/actions/pull/1002

@RobbieMcKinstry could we please have it behind us finally?

**Comment by RobbieMcKinstry** [comment]
Hello! I'm afraid I'm no longer a maintainer of this repository. Apologies, I wish I could help you.

**Pull request closed without merging**
